Jasmine/GameCommons/GameConstants.swift

```diff
@@ -2,17 +2,22 @@ import Foundation
 
 // swiftlint:disable nesting
 enum GameConstants {
+
+    /// Used to specify the level name in game title
+    static let formatSpecifierForString = "%@"
+
+    /// The time interval to run at for the game clock
     static let timeInterval: TimeInterval = 0.1
 
     enum Swapping {
         enum CiHui {
-            static let gameTitle = "Ci Hui (词汇) Swapping Game"
+            static let gameTitle = formatSpecifierForString + ": Ci Hui (词汇) Swapping Game"
```

Comment:
kinda no need to post-append "Ci Hui Swapping Game" and equivalent. but it's fine too.